qemu.go

```diff
@@ -483,7 +498,42 @@ func (q *qemu) setMemoryResources(podConfig PodConfig) ciaoQemu.Memory {
 		MaxMem: memMax,
 	}
 
-	return memory
+	return memory, nil
+}
+
+func getHostMemorySizeBytes(memInfoPath string) (uint64, error) {
+	f, err := os.Open(memInfoPath)
+	if err != nil {
+		return 0, err
+	}
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		// Expected format: ["MemTotal:", "1234", "kB"]
+		parts := strings.Fields(scanner.Text())
+
+		// Sanity checks: Skip malformed entries.
+		if len(parts) < 3 || parts[0] != "MemTotal:" || parts[2] != "kB" {
+			continue
+		}
+
+		// Convert kB to bytes.
+		size, err := strconv.Atoi(parts[1])
+		if err != nil {
+			continue
+		}
+		bytes := uint64(size) * 1024
```

Comment:
Why are we converting to bytes ? I mean we're basically multiplying by 1024 here to divide twice from the caller. I would not bother doing this and instead name the function `getHostMemorySizeKb`.